polylogue/cli/completion_engine.py

```
@@ -0,0 +1,282 @@
+            return self._option_completions("sync")
+        return []
+
+    def _complete_import(self, args: Sequence[str], current_index: int) -> List[Completion]:
+        if current_index == 1:
+            return [Completion(name) for name in ("chatgpt", "claude", "codex", "claude-code")]
+        prev = args[current_index - 1] if current_index > 0 else ""
+        if prev in {"--out", "--base-dir"}:
+            return [Completion("__PATH__")]
+        if current_index > 1 and args[1] in {"chatgpt", "claude"} and prev == "--source":
+            return [Completion("__PATH__")]
+        if current_word.startswith("--"):
```

Ruff (0.15.9)

[error] Undefined name `current_word`

(F821)